exercises/tournament/description.md

```diff
@@ -60,8 +60,3 @@ Devastating Donkeys;Courageous Californians;draw
 ```
 
 Means that the Devastating Donkeys and Courageous Californians tied.
-
-Your program should only process input lines that follow this format.
```

Comment:
My preference is to not add anything to the readme about what is tested.

I'd like to try the new readme that doesn't mention test input validity at all for a while and If it proves to be an issue in the future, someone can submit a PR to add the relevant language.